### PR TITLE
REG-14 update dbSNP reference

### PR DESCRIPTION
**Pull request by yunhailuo**

### CODE REVIEW

File: src/encoded/regulome_indexer.py [hunk 1]
```diff
@@ -258,9 +254,6 @@ def get_snp_index_mapping(chrom='chr1'):
                 'coordinates': {
                     'type': 'integer_range'
                 },
-                'strand': {
```
Comment (hitz) [inline src/encoded/regulome_indexer.py]: I guess SNPs don't have a strand?  Or maybe this code is left over from reading it from a BED

File: src/encoded/regulome_indexer.py [hunk 2]
```diff
@@ -904,7 +891,7 @@ def metadata_doc(afile, dataset):
             if dataset_type in dataset['@type']:
                 meta_doc['dataset_type'] = dataset_type
                 break
-        if afile.get('submitted_file_name') in SNP_FILES:
+        if dataset_type == 'Reference':
```
Comment (hitz) [inline src/encoded/regulome_indexer.py]: This seems a little sloppy; there are lots of references that might contain a parsable/indexable file that aren't SNPs.  Not sure I would worry about it now though.

File: src/encoded/regulome_indexer.py (hunk 1 above)
Comment (yunhailuo) [reply src/encoded/regulome_indexer.py]: Right. dbSNP data don't have strand info.

File: src/encoded/regulome_indexer.py (hunk 2 above)
Comment (yunhailuo) [reply src/encoded/regulome_indexer.py]: When it's here, it should already been filtered above with internal_tag (line 828-835). I only tag SNP reference as RegulomeDB. It's still not a very good solution. Any suggestions?

**Pull request merged**